Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 
 Bugfixes:
  * Assembler: Prevent incorrect calculation of tag sizes.
+ * EVM Assembly Import: Fix handling of missing source locations during import.
```

Comment:
Since we consider the import still experimental we could even have gone without the entry - but also not much harm in it!